runner/ansible/README.md

```diff
@@ -22,3 +28,102 @@ These are the fields needed by Trento:
 is that the labels are used for control purpose (select all the checks with this label e.g.), and the groups are used just for visual purposes
 - `description`: A longer description about the test purpose. It can be written using markdown.
 - `implementation`: Usually the task `main.yml` content
+
+## Check files
+
+The check files contain the actual task which the runner will execute. An
+example is available at [tasks/main.yml](roles/checks/1.1.1/tasks/main.yml).
+These files are written like a normal ansible task.
+
+## Check structure
+
+The checks folder is in `runner/ansible/roles/checks`. Each check has its own
+number (e.g. 1.1.1) which refers to its place in the queue (1.1.2 will be 
+executed after 1.1.1 etc.). Inside these numbered folders are two subfolders, both of these each has one file in them named `main.yml`:
+- `defaults`
+   The defaults directory (or better the main.yml file inside of it)
+   contains all the required [metadata](#metadata-files) for the check. 
+
+- `tasks`
+
+The checks themselfs are written like any ansible task, however their 
```

Comment:
I think all this description about the `task` should go in the `Check file` section

Reply:
Typo `themselfs -> themselves`